client-grpc/Cargo.toml

```diff
@@ -1,9 +1,8 @@
 [package]
-description = "Assets Service REST client"
+description = "Assets Service GRPC client"
 edition     = "2021"                                                       # Allowed values are 2015, 2018, 2021
 homepage    = "https://www.arrowair.com/docs/documentation/services/intro"
```

Comment:
use homepage.workspace = true and update the toplevel workspace Cargo.toml, see svc-storage's PR for example

same for all of the other attributes aside from "name", "description", and "version"

Reply:
[d227142](https://github.com/Arrow-air/svc-assets/pull/18/commits/d2271428712237b35575f5be1116dae5f0b0dd87)